nixos/modules/services/monitoring/netdata.nix

```diff
@@ -201,6 +201,8 @@ in {
       serviceConfig = {
         ExecStart = "${cfg.package}/bin/netdata -P /run/netdata/netdata.pid -D -c /etc/netdata/netdata.conf";
         ExecReload = "${pkgs.util-linux}/bin/kill -s HUP -s USR1 -s USR2 $MAINPID";
+        ExecPostStart = ''while [ "$(netdatacli ping)" != pong ]; do sleep 0.5; done'';
```

Comment:
`systemd[1]: /etc/systemd/system/netdata.service:29: Unknown key name 'ExecPostStart' in section 'Service', ignoring.`

It probably should be `ExecStopPost`, not `ExecPostStart`.

Reply:
:exploding_head: how did it work argh. I'll fix it.